### PR TITLE
Democracry consumer app refactors

### PR DESCRIPTION
**Pull request by smarshall-spitzbart**
# Description

Refactors to make the democ consumer app more understandable. Makes package aliases more accurate. Some alias' were prefixed with `ccv`, suggesting they came from this repo, when those packages were actually from the sdk repo.

This change will make PRs such as https://github.com/Stride-Labs/interchain-security/pull/1 easier to review, where the intricacies of package dependancies are important.

I believe this shouldn't affect production code at all, correct me if wrong

## Type of change

- [x] `Refactor`: Changes existing code style, naming, structure, etc.

## Regression tests

All existing tests do not fail. These changes should be harmless

### CODE REVIEW

File: app/consumer-democracy/app.go [hunk 1]
```diff
@@ -785,11 +786,6 @@ func (app *App) GetE2eEvidenceKeeper() e2e.E2eEvidenceKeeper {
 	return app.EvidenceKeeper
 }
 
-// GetE2eStakingKeeper implements the ConsumerApp interface.
```
Comment (smarshall-spitzbart) [inline app/consumer-democracy/app.go:788]: Unneeded, see [ConsumerApp](https://github.com/cosmos/interchain-security/blob/e39ba096963aeb965cbf4bd5059e0435eeb1fb51/testutil/e2e/interfaces.go#L45)

**Comment by smarshall-spitzbart** [comment]
Keeping as draft, as I think we're only merging changes to test files right now. Correct?

File: app/consumer-democracy/app.go [hunk 2]
```diff
@@ -82,30 +82,31 @@ import (
...
-	ccvmintkeeper "github.com/cosmos/cosmos-sdk/x/mint/keeper"
-	ccvminttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	mint "github.com/cosmos/cosmos-sdk/x/mint"
+	mintkeeper "github.com/cosmos/cosmos-sdk/x/mint/keeper"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
 
 	paramproposal "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
-	ibcconsumer "github.com/cosmos/interchain-security/x/ccv/consumer"
+	consumer "github.com/cosmos/interchain-security/x/ccv/consumer"
+	consumerkeeper "github.com/cosmos/interchain-security/x/ccv/consumer/keeper"
```
Comment (MSalopek) [inline app/consumer-democracy/app.go:107]:
```suggestion
             consumerkeeper "github.com/cosmos/interchain-security/x/ccv/consumer/keeper"
	ibcconsumerkeeper "github.com/cosmos/interchain-security/x/ccv/consumer/keeper"
```

These 2 are the same, possibly `ibcconsumerkeeper` is used in unchanged parts of the code which so they are not displayed in the diff

**Review by MSalopek — CHANGES_REQUESTED** [review]

File: app/consumer-democracy/app.go (hunk 2 above)
Comment (smarshall-spitzbart) [reply app/consumer-democracy/app.go:107]: Nice catch 👍, see https://github.com/cosmos/interchain-security/pull/679/commits/84794a9fe1f83b250947e57e03c72c8402a2b3f0

**Review by MSalopek — APPROVED** [review]

**Pull request merged**